mythril/ether/ethcontract.py

```diff
@@ -17,7 +17,8 @@ def __init__(self, code, creation_code="", name="Unknown", enable_online_lookup=
         code = re.sub(r'(_+.*_+)', 'aa' * 20, code)
 
         self.code = code
-        self.disassembly = Disassembly(self.code, enable_online_lookup=enable_online_lookup)
+        self.disassembly = Disassembly(code, enable_online_lookup=enable_online_lookup)
+        self.creation_disassemble = Disassembly(creation_code, enable_online_lookup=enable_online_lookup)
```

Comment:
An eth contract is used to model contracts that we don't have the construction code for, so we wouldn't need this here I think

Reply:
But SolidityContract takes the disassembly from ETHContract right?

Reply:
Yes, a solidity contract has creation code and is an ETHContract. The ethcontract baseclass takes case of the runtime disassembly. 
However the ETHContract is not a solidity contract and (generally) does not have creation code. The current implementation does have it sometimes, but by default it is ""

I do see that ETHContract already stored the creation_code, so your extension is not illogical.
Lets keep it like this for now